pkg/cloudprovider/providers/vsphere/vsphere_util.go

```diff
@@ -21,25 +21,30 @@ import (
 	"errors"
 	"os"
 	"regexp"
-	"runtime"
 	"strings"
 	"time"
 
 	"github.com/golang/glog"
-	"github.com/vmware/govmomi"
 	"github.com/vmware/govmomi/vim25"
 
 	"fmt"
 
 	"path/filepath"
 
 	"github.com/vmware/govmomi/vim25/mo"
-	"k8s.io/api/core/v1"
 	k8stypes "k8s.io/apimachinery/pkg/types"
 	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib"
 	"k8s.io/kubernetes/pkg/cloudprovider/providers/vsphere/vclib/diskmanagers"
 )
 
+const (
+	vSphereConfFileEnvVar = "VSPHERE_CONF_FILE"
+)
+
+var (
+	ConfFileLocation = os.Getenv(vSphereConfFileEnvVar)
```

Comment:
is this intentionally global so it can be mutated from other packages? that makes it hard to reason about side effects

Reply:
Nope. I have made it as a local variable. Please check the latest commit.